scripts/docker-compose/imports/keycloak/zaakafhandelcomponent-realm.json

```diff
@@ -479,6 +487,7 @@
         "zaakafhandelcomponent": [
           "domein_elk_zaaktype",
           "behandelaar",
+          "raadpleger",
```

Comment:
Order does not matter technically but visually maybe move the raadpleger above the behandelaar here to indicate that we have a hierarchy in these roles?